packages/kit-bg/src/providers/ProviderApiTon.ts

```
@@ -26,6 +26,7 @@ import type {
   SignDataRequest,
   SignProofRequest,
 } from '@onekeyfe/onekey-ton-provider';
+import { isNaN } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)